deployment/templates/daemonset.yaml

```diff
@@ -66,6 +66,9 @@ spec:
       - name: {{ .name | quote }}
         hostPath:
           path: {{ .hostPath | quote }}
+      - name: "exporter-metrics"
+        configMap:
+          name: "exporter-metrics-config-map"
```

Comment:
These lines are inside the range and will cause an error when installing the helm chart. They should be moved above or below.

Reply:
Thanks for heads up! Fixed.